### PR TITLE
Enable unused voice lines when sapping a robot

### PR DESCRIPTION
**Pull request by Thestickman391**
Might get overridden by TLK_STUNNED_TARGET but it didn't seem like it did in my testing

### CODE REVIEW

**Comment by Yttrium-tYcLief** [comment]
Out of curiosity, are these voice lines available to hear anywhere, like on the wiki?

**Comment by Thestickman391** [comment]
https://youtu.be/DyK8SKbRQho?t=95

**Comment by Yttrium-tYcLief** [comment]
Interesting. Are you only using the second half of the lines? The first half sounds like the class is doing it themselves, which isn't possible.

File: game/server/tf/tf_obj_sapper.cpp [hunk 1]
```diff
@@ -801,6 +801,7 @@ void CObjectSapper::ApplyRoboSapper( CTFPlayer *pTarget, float flDuration, int n
 	if ( IsValidRoboSapperTarget( pTarget ) )
 	{
 		ApplyRoboSapperEffects( pTarget, flDuration );
+		pPlayer->SpeakConceptIfAllowed( MP_CONCEPT_MVM_SAPPED_ROBOT );
```
Comment (mastercoms) [inline game/server/tf/tf_obj_sapper.cpp:804]: Should this check if they're stealthed?

**Review by mastercoms — CHANGES_REQUESTED** [review]

File: game/server/tf/tf_obj_sapper.cpp (hunk 1 above)
Comment (Thestickman391) [reply game/server/tf/tf_obj_sapper.cpp:804]: I think SpeakConceptIfAllowed shouldn't allow speaking while cloaked. Regardless, you can't sap while cloaked anyway so hopefully this isn't an issue? If it is, we can do what I did in #132 and put it in game/server/tf/tf_player.cpp instead.

**Comment by KazerGazer** [comment]
> 
> 
> Interesting. Are you only using the second half of the lines? The first half sounds like the class is doing it themselves, which isn't possible.

I think you're only supposed to hear them as a Spy disguised as one of the classes, like how saying a voice line while disguised will have the Spy hear the voice of the class they're disguised as, while his teammates will hear the voice line as the Spy's voice. I don't think Spy actually has any voice lines talking about sapping, though.

**Pull request closed without merging**